int16 finger_id
---
float64[24] jacobian